app/models/webhook.rb

```diff
@@ -16,11 +16,14 @@ class Webhook < ApplicationRecord
 
   DEFAULT_PAYLOAD_TEMPLATE = 'Webhook Payload - Payload Default'
 
+  ALLOWED_HTTP_METHODS = %w[POST GET PUT DELETE PATCH].freeze
+
   attribute :events, :string, array: true, default: []
 
   validates_lengths_from_database
   validates :name, :target_url, :events, presence: true
   validates :target_url, format: URI.regexp(%w[http https])
```

Comment:
Nit: Do we want to improve the validator here? Does this give a meaningful message?